packages/storage/test/unit/service.compat.test.ts

```diff
@@ -187,7 +187,7 @@ describe('Firebase Storage > Service', () => {
       }, 'storage/invalid-default-bucket');
     });
   });
-  describe('connectStorageEmulator(service, host, port)', () => {
+  describe('connectStorageEmulator(service, host, port, options)', () => {
```

Comment:
Can you update the tests in `service.exp.test.ts` as well? You probably only want to add new tests to `service.exp.test.ts`, because we are removing redundant compat tests in https://github.com/firebase/firebase-js-sdk/pull/5271

Reply:
Sure, moved